tests/test_simple_range_query.py

```diff
@@ -1,29 +1,42 @@
 import pandas as pd
+import pytest
 from pandas.testing import assert_frame_equal
 
 from fsql.api import read_partitioned_table
-from fsql.query import ColumnComparator, ColumnRange, SimpleRangeQuery
+from fsql.query import Q_AND, Q_EQ, Q_OR, ColumnComparator, ColumnRange, SimpleRangeQuery
+
+
+def test_invalid_ranges():
+    [
+        ColumnRange("any", 1, 3, ColumnComparator.num),
+        ColumnRange("any", "a", "a"),
+    ]
+    with pytest.raises(ValueError):
```

Comment:
Nit: I think it's better to add `match="invalid range"` (eventually we could introduce `InvalidRangeError`, but it's not necessary I think)